README.adoc

```diff
@@ -341,8 +357,13 @@ dependencyManagement {
     }
 }
 ----
-This options ensures that `4.0.51.Final` will be used only for `gatling` configurations, leaving other dependencies unchanged.
+
+These options ensure that `4.0.51.Final` will be used only for `gatling` configurations, leaving other dependencies unchanged.
 
 == Release a new version
 
 . Push a tag `vX.Y.Z`, where `X.Y.Z` is the version to publish.
+
+== Acknowledgements
```

Comment:
You must mention @lkishalmi as a maintainer and current plugin repository owner.